app/models/user.rb

```diff
@@ -43,7 +45,16 @@ def minor
   end
 
   def insufficient_credit
-    provider == 'amber_oauth2' and credit < -5
+    provider == 'amber_oauth2' and credit < 0
```

Comment:
In the past it was 'gedoogd' to be negative up to 5 euros. With the new protocol that is no longer the case.